pkg/pillar/containerd/oci.go

```diff
@@ -535,3 +531,24 @@ func (s *ociSpec) UpdateEnvVar(envVars map[string]string) {
 		}
 	}
 }
+
+// UpdateShimVMDevices grants full access to file devices for shim VM OCI specs
+func (s *ociSpec) UpdateShimVMDevices() {
+	if len(s.Get().Process.Args) == 0 {
+		return
+	}
+
+	ctrCmd := s.Get().Process.Args[0]
```

Comment:
please check, that `Get()` does not return `nil`, that `Process` is not `nil` and that `len(Args) > 0`.